security/qatlib_scc.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Can we change the commit log to:

Security: Adding intel-qat-scc for Intel QAT technology-based workload

This user-defined SCC is based on the [OCP predefined restricted-v2 SCC](https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth)
The IPC_LOCK capability and RunAsAny permission are added to the restricted-v2 SCC for intel-qat-scc

This patch is used to fix https://github.com/intel/intel-technology-enabling-for-openshift/issues/122

Signed-off-by: vbedida79 <veenadhari.bedida@intel.com>

Reply:
can we also change the PR name to
Security: Adding intel-qat-scc for Intel QAT technology-based workload